classes/SuperDirt.sc

```diff
@@ -53,10 +54,14 @@ SuperDirt {
 
 	init {
 		soundLibrary = DirtSoundLibrary(server, numChannels);
+		outputvolume = server.volume;
+		outputvolume.setVolumeRange(-90, 0);
```

Comment:
We want to be able to use SuperDirt with other libraries – we shouldn't mess with the server volume, I think.

Reply:
I am sure there is another way to set volume? Normally we are in the range [min = -90, max = 6] so here you have found a reason why this should be less?